apps/perf-test/package.json

```diff
@@ -24,6 +24,7 @@
     "lodash": "^4.17.15"
   },
   "dependencies": {
+    "@fluentui/react-avatar": "^0.6.0",
```

Comment:
This should be on the latest version (`^0.6.3`) in all packages. Once you fix it in one package you should be able to run `yarn syncpack:fix` to automatically fix the rest.

Reply:
Ok. I had reverted the package version to match what was checked into the examples package so I wouldn't be modifying unrelated packages in this PR. But I'll go back and update that other package too.

Reply:
Hmm if the examples package is different then @khmakoto or I must have had a bad merge at some point. I'll fix that in master, sorry.

Reply:
It's fixed now. Apologies again about requiring the extra changes. :(

Reply:
Ok thanks -- just pulled in your change